notebooks/py/ISS.py

```diff
@@ -56,7 +56,8 @@
 # EPY: START markdown
 ### Visualize Codebook
 #
-#The ISS codebook maps each barcode to a gene.This protocol asserts that genes are encoded with a length 4 quatenary barcode that can be read out from the images. Each round encodes a position in the codeword. The maximum signal in each color channel (columns in the above image) corresponds to a letter in the codeword. The channels, in order, correspond to the letters: 'T', 'G', 'C', 'A'. 
+#The ISS codebook maps each barcode to a gene. This protocol asserts that genes are encoded with
```

Comment:
You should unwrap this.  Exception to our normal 100col limit, because it makes the jupyter notebook render oddly.